.github/workflows/release.yml

```diff
@@ -0,0 +1,51 @@
+name: release
+
+on: workflow_dispatch 
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      # setup
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+
+      # build resources
+      - run: node ./dev/icon_generator/makeicons
+      - run: node ./dev/resource_packager
+
+      # publish
+      # TODO: cleaner version number
+      - name: get version
+        id: version
+        run: |
+          echo ::set-output name=current-version::$(cat ./editor/script/engine/bitsy.js | grep -oP '(?<=major: )\d+').$(cat ./editor/script/engine/bitsy.js | grep -oP '(?<=minor: )\d+')
+      - name: publish to itch.io
+        uses: josephbmanley/butler-publish-itchio-action@v1.0.3
+        env:
+          BUTLER_CREDENTIALS: ${{ secrets.BUTLER_CREDENTIALS }}
+          CHANNEL: win-linux-mac-stable
```

Comment:
using "win", "linux", and "mac" in this channel name is semantic, and will tell itch to automatically check the "works on [platform]" flags ([reference](https://itch.io/docs/butler/pushing.html#channel-names))

since this is a web project this doesn't matter much but might as well include it

Reply:
makes sense!